indico/testing/fixtures/user.py

```diff
@@ -60,3 +60,15 @@ def _create_group(id_):
 def dummy_group(create_group):
     """Create a mocked dummy group."""
     return create_group(1337)
+
+
+@pytest.fixture
+def create_identity(db):
+    """Return a callable which lets you create dummy identities."""
+    def _create_identity(user, provider, identifier):
+        identity = Identity(user_id=user.id, provider=provider, identifier=identifier)
```

Comment:
```suggestion
        identity = Identity(user=user, provider=provider, identifier=identifier)
```

It's always better to use the relationship attributes instead of the foreign keys. That way you most likely don't even need the `db.session.add()` call below since the user is already in the DB session and this automatically chains across relationships.

Reply:
That's very true. The updated commits implement the suggested change. However, I tested that indeed `db.session.add()` is not needed, as you expected. 
Should I also drop this?

Reply:
yeah I'd drop it

Reply:
Done and force-pushed :smile:.